### PR TITLE
Ignore non HTTP(S) WARC records

### PR DESCRIPTION
**Pull request by benoit74**
Might have been a solution to #332 but not (see next comment)

NOTA : there is no test because I failed to find under which conditions we can have a non HTTP(S) WARC record + we should test for all places where we iterate on WARC records, which makes it even harder to test

### CODE REVIEW

**Comment by benoit74** [comment]
This is not fixing #332 in fact, I found why we have these weird URL schemes, bug is same as #356 

I propose to still merge this PR, it is valuable, should the WARC contain records which are not http/https we still need to ignore them.

File: src/warc2zim/converter.py [hunk 1]
```diff
@@ -397,7 +397,12 @@ def gather_information_from_warc(self):
             if record.rec_type not in ("response", "revisit"):
                 continue
 
-            url = get_record_url(record)
+            url = str(get_record_url(record))
```
Comment (rgaudin) [inline src/warc2zim/converter.py:400]: `get_record_url()`  has no return type but can it ever return other than an `str`? Wouldn't it be better then to set the return type?

**Review by rgaudin — APPROVED** [review]
OK

File: src/warc2zim/converter.py (hunk 1 above)
Comment (benoit74) [reply src/warc2zim/converter.py:400]: Agreed

**Pull request merged**